### PR TITLE
Revert hard coded Name string in templates

### PR DESCRIPTION
**Pull request by lordelix**
## Description
Revert back hard-coded name string in core template files

## Related Issue

https://github.com/owncloud/enterprise/issues/2271

## How Has This Been Tested?

Visual test

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/files_sharing/templates/public.php [hunk 1]
```diff
@@ -56,7 +56,7 @@
 		 data-owner-display-name="<?php p($_['displayName']) ?>" data-owner="<?php p($_['owner']) ?>" data-name="<?php p($_['filename']) ?>">
 		<a href="<?php print_unescaped(link_to('', 'index.php')); ?>" title="" id="owncloud">
 			<h1 class="logo-icon">
-				<span class="own">own</span><span class="cloud">Cloud</span>
+				<?php $theme->getName(); ?>
```
Comment (felixheidecke) [inline apps/files_sharing/templates/public.php]: This, unfortunately, overrides CE & EE theme. Shouldn't happen but does :-1:

File: lib/private/legacy/defaults.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ function __construct() {
 		$version = \OCP\Util::getVersion();
 
 		$this->defaultEntity = 'ownCloud'; /* e.g. company name, used for footers and copyright notices */
-		$this->defaultName = 'ownCloud'; /* short name, used when referring to the software */
+		$this->defaultName = '<span class="own">own</span><span class="cloud">Cloud</span>'; /* short name, used when referring to the software */
```
Comment (felixheidecke) [inline lib/private/legacy/defaults.php]: To provide the look needed in the EE Theme, we a bit more HTML in the defaultName. (Related to https://github.com/owncloud/core/pull/29161/files#r143151177)

This should not be escaped but render out valid HTML. @patrickjahns or @DeepDiver1975

**Review by michaelstingl — CHANGES_REQUESTED** [review]
Please check my comment in https://github.com/owncloud/enterprise/issues/2271#issuecomment-334782490

File: lib/private/legacy/defaults.php (hunk 2 above)
Comment (patrickjahns) [reply lib/private/legacy/defaults.php]: 👎 I am not comfortable adding HTML tags to a string only field

**Comment by PVince81** [comment]
Fixes https://github.com/owncloud/core/issues/29172

**Comment by felixheidecke** [comment]
I removed the hard-coded ownCloud occurrences in core.
This will alter the look in the files_sharing app in conjunction with the enterprise theme, but I guess we can live with that ;-)

@patrickjahns

**Comment by DeepDiver1975** [comment]
please squash ...

**Comment by felixheidecke** [comment]
Please review @DeepDiver1975 & @patrickjahns … No longer in progress.

**Review by patrickjahns — APPROVED** [review]
LGTM

@michaelstingl has the final say in this matter

**Review by michaelstingl — APPROVED** [review]
LGTM, but I only tested the ownBrander generated theme and I didn't test if this breaks other stuff… (vanilla oC, example theme etc)

**Comment by PVince81** [comment]
@felixheidecke please backport to stable10

**Pull request merged**